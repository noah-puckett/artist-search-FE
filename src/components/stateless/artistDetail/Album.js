import React from 'react';
import PropTypes from 'prop-types';

function Album({ title }) {
    return (<>
        <h4>{title}</h4>
    </>
    );
}

Album.propTypes = {
    title: PropTypes.string.isRequired,
};

export default Album;
