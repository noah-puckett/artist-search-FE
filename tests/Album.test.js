import React from 'react';
import { shallow } from 'enzyme';
import Album from '../src/components/stateless/artistDetail/Album';

describe('Album component', () => {
    it('renders Album', () => {
        const wrapper = shallow(<Album title='hello' />);
        expect(wrapper).toMatchSnapshot();
    });
});
