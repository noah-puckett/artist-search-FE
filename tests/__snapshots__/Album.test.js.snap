// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Album component renders Album 1`] = `
<Fragment>
  <h4>
    hello
  </h4>
</Fragment>
`;
